Dockerfile

```diff
@@ -1,11 +1,14 @@
-FROM golang:1.11-alpine
+FROM golang:1.12-alpine
 # This container is for daily development.
 
 RUN apk add --no-cache build-base git libseccomp-dev linux-headers
-RUN go get github.com/seccomp/libseccomp-golang && \
-    go get github.com/fatih/color && \
-    go get github.com/gobwas/glob && \
-    go get gopkg.in/yaml.v2
+#RUN go get github.com/seccomp/libseccomp-golang && \
+#    go get github.com/fatih/color && \
+#    go get github.com/gobwas/glob && \
+#    go get gopkg.in/yaml.v2
+
+# Only one CMD command, so change this to RUN
+# CMD ["make", "inside-container"]
```

Comment:
Where do you run this command in your new build pipeline?
My intention was that if you run `make manylinux` or `make musllinux`, then there should be the compiled binary upon completion, by running this command by default when you `docker run` the resulting image.

Reply:
I see. My intention was to build the binary inside the container using `make build`. I am not using `inside-container` as well. But I can fix it according to what you said.